src/System.Drawing.Common/tests/Graphics_DrawBezierTests.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Drawing.Tests
+{
+    public class Graphics_DrawBezierTests : DrawingTest
+    {
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawBezier_Point()
+        {
+            using (Bitmap image = new Bitmap(100, 100))
+            using (Pen pen = new Pen(Color.White))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                graphics.DrawBezier(pen, new Point(10, 10), new Point(20, 1), new Point(35, 5), new Point(50, 10));
+                ValidateImageContent(image,
+                    PlatformDetection.IsWindows
+                        ? new byte[] { 0xa4, 0xb9, 0x73, 0xb9, 0x6f, 0x3a, 0x85, 0x21, 0xd3, 0x65, 0x87, 0x24, 0xcf, 0x6d, 0x61, 0x94 }
+                        : new byte[] { 0xcf, 0x92, 0xaa, 0xe2, 0x44, 0xd4, 0xdd, 0xae, 0xdd, 0x4c, 0x8a, 0xf5, 0xc3, 0x65, 0xac, 0xf2 });
+            }
+        }
+
+        [ConditionalFact(Helpers.GdiplusIsAvailable)]
+        public void DrawBezier_Points()
+        {
+            using (Bitmap image = new Bitmap(100, 100))
+            using (Pen pen = new Pen(Color.Red))
+            using (Graphics graphics = Graphics.FromImage(image))
+            {
+                Point[] points =
+                {
+                    new Point(10, 10), new Point(20, 1), new Point(35, 5), new Point(50, 10),
+                    new Point(60, 15), new Point(65, 25), new Point(50, 30)
+                };
+
+                graphics.DrawBeziers(pen, points);
+                ValidateImageContent(image,
+                    PlatformDetection.IsWindows
```

Comment:
Just curious, why is `IsWindows` needed if this test is already conditioned on GDI+ availability? What is the other platform?

Reply:
The check for GDI+ availability is also checking for the mono version of GDI+, which we use on Unix. The implementation isn't exactly the same and has some gaps.